notify/impl.go

```diff
@@ -691,6 +695,134 @@ func (n *Hipchat) retry(statusCode int) (bool, error) {
 	return false, nil
 }
 
+//Wechat implements a Notfier for wechat notifications
```

Comment:
There's a space between `//` and the first word, so it should be `// Wechat ...`